server/main.lua

```diff
@@ -37,7 +37,9 @@ RegisterNetEvent('qb-vehiclekeys:server:breakLockpick', function(itemName)
 end)
 
 RegisterNetEvent('qb-vehiclekeys:server:setVehLockState', function(vehNetId, state)
-    SetVehicleDoorsLocked(NetworkGetEntityFromNetworkId(vehNetId), state)
+    local vehicle = NetworkGetEntityFromNetworkId(vehNetId)
+    SetVehicleDoorsLocked(vehicle, state)
```

Comment:
Is this still needed if the statebag is going to sync all the clients?

Reply:
The answer is yes, when a player enters the scope of an entity, the statebag change handler is triggered, even if the value was set at some time in the past.